samples/msal-browser-samples/VanillaJSTestApp2.0/app/inMemoryStorage/authConfig.js

```diff
@@ -0,0 +1,55 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "3fba556e-5d4a-48e3-8e1a-fd57c12cb82e",
+        authority: "https://login.windows-ppe.net/common/"
+    },
+    cache: {
+        cacheLocation: "inMemoryStorage", // This configures where your cache will be stored
```

Comment:
```suggestion
        cacheLocation: "memoryStorage", // This configures where your cache will be stored
```

Reply:
Good example of where the enum would be helpful

Reply:
done.